manim/scene/scene.py

```diff
@@ -104,9 +104,11 @@ def __str__(self):
     def print_end_message(self):
         """
         Used internally to print the number of
-        animations played after the scene ends.
+        animations played after the scene ends, 
+        as well as the name of the scene rendered 
+        (useful when using the `-a` option).
         """
-        logger.info("Played {} animations".format(self.num_plays))
+        logger.info(f"Rendered {self.__str__()}\nPlayed {self.num_plays} animations")
```

Comment:
Question: any reason to use `self.__str__` instead of `str(self)`?

Reply:
Not really.... Thought they do the same thing? Also a bit easier to find out where the name comes from since `Scene` has a custom `__str__` dunder :

https://github.com/ManimCommunity/manim/blob/401577cf1fcfc94912f298f0a1cac775fbd8283f/manim/scene/scene.py#L101-L102

which I think might be a bit harder to notice if we use `str(self)`.

I'll change it if you want though.

Reply:
If they do the same thing, let's change it. The whole point of magic dunder methods in python is that you never have to call them yourself.

Reply:
Changed in latest commit.